CHANGELOG.md

```diff
@@ -1,6 +1,29 @@
 # Changelog
 
-## 2020-07-03 - 1.3.0
+All notable changes to this project will be documented in this file.
+Each new release typically also includes the latest modulesync defaults.
+These should not affect the functionality of the module.
+
+## [1.4.0](https://github.com/voxpupuli/modulesync/tree/1.4.0) (2020-08-12)
+
+[Full Changelog](https://github.com/voxpupuli/modulesync/compare/1.3.0...1.4.0)
+
+**Implemented enhancements:**
+
+- Symbolize keys in managed\_modules except for module names [\#185](https://github.com/voxpupuli/modulesync/pull/185) ([raphink](https://github.com/raphink))
+
+**Fixed bugs:**
+
+- GitLab MR: undefined method `\[\]' for nil:NilClass \(NoMethodError\) [\#187](https://github.com/voxpupuli/modulesync/issues/187)
+- msync fails with nilClass error [\#172](https://github.com/voxpupuli/modulesync/issues/172)
+- Fix NoMethodError for --pr option \(caused by `module_options = nil`\) [\#188](https://github.com/voxpupuli/modulesync/pull/188) ([bittner](https://github.com/bittner))
+- Allow empty module options in self.pr\(\) [\#186](https://github.com/voxpupuli/modulesync/pull/186) ([raphink](https://github.com/raphink))
+
+**Closed issues:**
+
+- Remove file namespaces from configuration [\#27](https://github.com/voxpupuli/modulesync/issues/27)
```

Comment:
The last updates suggest it was resolved a while back. Do we want to include this?